test/post_comment_handler_test.py

```diff
@@ -0,0 +1,232 @@
+                          "Expected size of comment's list should be zero")
+
+        # Call the post method
+        self.testapp.post("/api/post/%s/comment" %
+                          self.mayza_post.key.urlsafe(),
+                          json.dumps(self.other_comment))
+
+        # Update post
+        self.mayza_post = self.mayza_post.key.get()
+
+        # Verify size of list
+        self.assertEquals(len(self.mayza_post.comments), 1,
+                          "Expected size of comment's list should be one")
+
+    def test_delete(self):
+        """User can delete your comment in Post."""
+        # Pretend an authentication
+        self.os.environ['REMOTE_USER'] = 'maiana.brito@ccc.ufcg.edu.br'
+        self.os.environ['USER_EMAIL'] = 'maiana.brito@ccc.ufcg.edu.br'
+
+        # Added comment
+        self.response = self.testapp.post(
+            "/api/post/%s/comment" % self.mayza_post.key.urlsafe(),
+            json.dumps(self.comment)).json
+        # ID of comment
+        self.id_comment = self.response["id"]
+        self.mayza_post = self.mayza_post.key.get()
+        self.assertEquals(len(self.mayza_post.comments), 1,
+                          "Expected size of comment's list should be one")
+
+        # Call the delete method
+        self.testapp.delete("/api/post/%s/comment/%s" %
+                            (self.mayza_post.key.urlsafe(), self.id_comment))
+
+        # Update post
+        self.mayza_post = self.mayza_post.key.get()
+
+        # Verify size of list
+        self.assertEquals(len(self.mayza_post.comments), 0,
+                          "Expected size of comment's list should be zero")
+
+    def test_delete_simpleuser(self):
+        """An simple user can't delete comments by other users in Post."""
+        # Pretend an authentication
+        self.os.environ['REMOTE_USER'] = 'mayzabeel@gmail.com'
+        self.os.environ['USER_EMAIL'] = 'mayzabeel@gmail.com'
+
+        # Added comment of Mayza
+        self.response = self.testapp.post(
+            "/api/post/%s/comment" % self.mayza_post.key.urlsafe(),
+            json.dumps(self.comment)).json
+        # ID of comment
+        self.id_comment = self.response["id"]
+        self.mayza_post = self.mayza_post.key.get()
+        self.assertEquals(len(self.mayza_post.comments), 1,
+                          "Expected size of comment's list should be one")
+
+        # Pretend an authentication
+        self.os.environ['REMOTE_USER'] = 'maiana.brito@ccc.ufcg.edu.br'
+        self.os.environ['USER_EMAIL'] = 'maiana.brito@ccc.ufcg.edu.br'
+
+        # User Maiana call the delete method
+        with self.assertRaises(Exception) as ex:
+            self.testapp.delete("/api/post/%s/comment/%s" %
+                                (self.mayza_post.key.urlsafe(), self.id_comment))
+
+        ex = get_message_exception(self, ex.exception.message)
+        self.assertEquals(ex, "Error! User not allowed to remove comment")
```

Comment:
Nesse coloca que permanece 1
self.assertEquals(len(self.mayza_post.comments), 1,
                           "Expected size of comment's list should be one")